compiler/env/OMRObjectModel.hpp

```diff
@@ -110,12 +111,43 @@ class ObjectModel
    /**
    * @brief: Returns true if concurrent scavenging enabled in the VM's GC
    */
-   bool shouldGenerateReadBarriersForFieldLoads() { return false; };
+   bool shouldGenerateReadBarriersForFieldLoads() { return false; }
 
    /**
    * @brief: Returns true if option for software read barriers is enabled in the VM's GC
    */
-   bool shouldReplaceGuardedLoadWithSoftwareReadBarrier() { return false; };
+   bool shouldReplaceGuardedLoadWithSoftwareReadBarrier() { return false; }
+
+   /**
+    * \brief Dereference a static reference field
+    *
+    * \param
+    *     comp TR::Compilation
```

Comment:
Perhaps a better comment is needed here as to why this parameter is necessary or what it may be used for?